distribution/lib/Standard/Table/0.0.0-dev/src/Table.enso

```diff
@@ -151,13 +151,14 @@ type Table
         invalid_input_shape =
             Error.throw (Illegal_Argument.Error "Each column must be represented by a vector whose first element is the column name, second element is a vector of elements that will constitute that column (or an existing column) and an optional third element that is the Value_Type for the column. Got: "+columns.to_text)
         cols = columns.map on_problems=No_Wrap.Value c->
-            column = case c of
+            column_with_warning = case c of
                 v : Vector -> case v.length of
                     2 -> Column.from_vector (v.at 0) (v.at 1)
                     3 -> Column.from_vector (v.at 0) (v.at 1) (v.at 2)
                     _ -> invalid_input_shape
                 col : Column -> col
                 _ -> invalid_input_shape
+            column = Warning.clear column_with_warning
```

Comment:
Workaround for https://github.com/enso-org/enso/pull/13118#issuecomment-2895098807